plugin-flex-ts-template-v2/src/feature-library/send-audio-rec-file/custom-components/AudioRecorderPanel/AudioRecorderPanel.Styles.tsx

```diff
@@ -0,0 +1,15 @@
+import { styled } from '@twilio/flex-ui';
+
+// import styledCss, { keyframes, css } from 'styled-components';
```

Comment:
Can remove the commented out import here.

Reply:
comment removed.
Done